### ISSUE TITLE
Should accountId be necessary for sending data?

### ISSUE THREAD

**Issue by arkocal** [Open-IoT-Service-Platform/oisp-frontend#12]
Requests to endpoint `/data/{deviceId}` require the accountId in the payload. Because only activated devices can send data and such devices are connected to a single account, this seems uncessary to me, especially considering accountId can be retrieved from device token(?) .

**Comment by arkocal** [comment]
@wagmarcel You were recently working on data sending on frontend. Are we any wiser about this now?

**Comment by wagmarcel** [comment]
@arkocal you are right, this information is redundant. But besides that, I don't see any security implications and thus, no priority to change that.
